src/Providers.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
```

Comment:
Let's use this way of importing and then extending not 'React.PureComponent' but 'PureComponent' everywhere else too.